raiden-dapp/src/components/ConfirmationDialog.vue

```diff
@@ -1,95 +1,67 @@
 <template>
-  <v-row class="confirmation" no-gutters>
-    <v-row align="center" justify="center">
-      <h1 class="confirmation__text__header"><slot name="header"></slot></h1>
-    </v-row>
-    <v-row align="center" justify="center">
-      <v-col cols="12">
-        <p class="confirmation__text__message"><slot></slot></p>
-      </v-col>
-    </v-row>
-    <v-row
-      align="end"
-      justify="center"
-      class="confirmation__buttons"
-      no-gutters
-    >
-      <v-btn
-        :id="`cancel-${identifier}`"
-        class="text-capitalize confirmation__buttons__cancel"
-        @click="cancel()"
-      >
-        {{ $t('confirmation.buttons.cancel') }}
-      </v-btn>
-      <v-btn
-        :id="`confirm-${identifier}`"
-        class="text-capitalize confirmation__buttons__confirm"
-        @click="confirm()"
-      >
-        {{ positiveAction }}
-      </v-btn>
-    </v-row>
-  </v-row>
+  <raiden-dialog
+    class="confirmation"
+    :visible="visible"
+    :fullscreen="false"
+    @close="cancel"
+  >
+    <v-card-title>
+      <v-row align="center" justify="center">
+        <v-col>
+          <span>
+            <slot name="header"></slot>
+          </span>
+        </v-col>
+      </v-row>
+    </v-card-title>
+
+    <v-card-text>
+      <v-row align="center" justify="center">
+        <span>
+          <slot></slot>
+        </span>
+      </v-row>
+    </v-card-text>
+
+    <v-card-actions>
+      <v-row align="center" justify="center">
+        <v-col>
+          <action-button
+            :id="`confirm-${identifier}`"
+            :enabled="true"
+            :text="positiveAction"
+            @click="confirm()"
+          >
+          </action-button>
+        </v-col>
+      </v-row>
+    </v-card-actions>
+  </raiden-dialog>
 </template>
 
 <script lang="ts">
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator';
-@Component({})
-export default class Confirmation extends Vue {
+import ActionButton from '@/components/ActionButton.vue';
+import RaidenDialog from '@/components/RaidenDialog.vue';
+@Component({
+  components: {
+    RaidenDialog,
+    ActionButton
+  }
+})
+export default class ConfirmationDialog extends Vue {
   @Prop({ required: true })
-  identifier!: number;
+  visible!: boolean;
   @Prop({ required: true })
   positiveAction!: string;
+  @Prop({ required: true })
+  identifier!: number;
+
   @Emit()
-  public cancel() {}
+  cancel() {}
   @Emit()
   public confirm() {}
```

Comment:
nitpick: since we are removing the public property from cancel maybe we should remove it from confrirm too.